paddle/fluid/operators/multinomial_op.cu

```diff
@@ -31,6 +32,16 @@ __global__ void NormalizeProbability(T* norm_probs, const T* in_data,
                                      T* sum_rows) {
   int id = threadIdx.x + blockIdx.x * blockDim.x +
            blockIdx.y * gridDim.x * blockDim.x;
+  PADDLE_ENFORCE(
+      in_data[id] >= 0.0,
+      "The input of multinomial distribution should be >= 0, but got %f",
+      in_data[id]);
+  PADDLE_ENFORCE(in_data[id] != INFINITY,
+                 "The input of multinomial distribution shoud not be infinity");
+  PADDLE_ENFORCE(in_data[id] != NAN,
+                 "The input of multinomial distribution shoud not be NaN");
+  PADDLE_ENFORCE(sum_rows[blockIdx.y] > 0.0,
+                 "The sum of input should not be 0");
```

Comment:
Do you mean >0 here?

Reply:
Yes. It's `> 0`, and `>0` has the same meaning with `not be 0` here. Because `<0` has been forbidden before.
I have change the description from `not be 0` to `>0`.